test/functions.jl

```diff
@@ -3,6 +3,15 @@
     x = MOI.VariableIndex(1)
     y = MOI.VariableIndex(2)
     z = MOI.VariableIndex(3)
+    @testset "VectorAffineTerm/VectorQuadraticTerm construction" begin
+        NonNativeInt = Int == Int32 ? Int64 : Int32
```

Comment:
It's sufficient to just test with `Int32`. Native vs. nonnative is unneeded complexity in this context.

Reply:
Addressed.